pimd/pim_iface.h

```diff
@@ -93,7 +97,7 @@ struct pim_secondary_addr {
 
 struct pim_interface {
 	uint32_t options; /* bit vector */
-	ifindex_t mroute_vif_index;
+	ifindex_t mroute_if_index;
```

Comment:
This name change is really confusing. Ifindex and vif index are 2 separate things. we should not rename to if_index.